sys/aarch64/trap.c

```diff
@@ -75,11 +75,9 @@ static void abort_handler(ctx_t *ctx, register_t esr, vaddr_t vaddr,
   if (error == EACCES || error == EINVAL)
     goto fault;
 
-  vm_map_t *vmap = vm_map_lookup(vaddr);
-  if (!vmap) {
-    klog("No virtual address space defined for %lx!", vaddr);
-    goto fault;
-  }
+  vm_map_t *vmap = vm_map_user();
+  assert(vmap);
```

Comment:
Now every invocation of `vm_map_user` expects non-null return value. Let's move assert to `vm_map_user`.